content/en/docs/concepts/staged-update.md

```diff
@@ -232,34 +265,92 @@ metadata:
-  resourceSnapshotIndex: "5"                    # Resource version to deploy
-  stagedRolloutStrategyName: app-rollout-strategy # Strategy to execute
+  placementName: example-namespace-placement      # Required: Target ResourcePlacement
+  resourceSnapshotIndex: "5"                      # Optional: Resource version (omit for latest)
+  stagedRolloutStrategyName: app-rollout-strategy # Required: Strategy to execute
+  state: Initialize                               # Optional: Initialize (default), Run, or Stop
+```
+
+**Using Latest Resource Snapshot:**
+```yaml
+apiVersion: placement.kubernetes-fleet.io/v1beta1
+kind: ClusterStagedUpdateRun
+metadata:
+  name: example-run-latest
+spec:
+  placementName: example-placement
+  # resourceSnapshotIndex omitted - system uses latest snapshot automatically
+  stagedRolloutStrategyName: example-strategy
+  state: Run
+```
+
+### UpdateRun State Management
+
+UpdateRuns support three states to control execution lifecycle:
+
+| State | Behavior | Use Case |
+|-------|----------|----------|
+| **Initialize** | Prepares the updateRun without executing (default) | Review computed stages before starting rollout |
+| **Run** | Executes the rollout or resumes from stopped state | Start or resume the staged rollout |
+| **Stop** | Pauses execution at current cluster/stage | Temporarily halt rollout for investigation |
+
+**Valid State Transitions:**
+- `Initialize` → `Run`: Start the rollout
+- `Run` → `Stop`: Pause the rollout
+- `Stop` → `Run`: Resume the rollout
+
+**Invalid State Transitions:**
+- `Initialize` → `Stop`: Cannot stop before starting
+- `Run` → `Initialize`: Cannot reinitialize after starting
+- `Stop` → `Initialize`: Cannot reinitialize after stopping
+
+The `state` field is the **only mutable field** in the UpdateRunSpec. You can update it to control rollout execution:
+
+```bash
+# Start a rollout
+kubectl patch csur example-run --type='merge' -p '{"spec":{"state":"Run"}}'
+
+# Pause a rollout
+kubectl patch csur example-run --type='merge' -p '{"spec":{"state":"Stop"}}'
+
+# Resume a paused rollout
+kubectl patch csur example-run --type='merge' -p '{"spec":{"state":"Run"}}'
 ```
 
 ### UpdateRun Execution
 
 UpdateRuns execute in two phases:
-1. **Initialization**: Captures strategy snapshot, collects target bindings, generates cluster update sequence
-2. **Execution**: Processes stages sequentially, updates clusters within each stage, enforces after-stage tasks
+1. **Initialization**: Captures strategy snapshot, collects target bindings, generates cluster update sequence. Occurs when state is `Initialize` or `Run`
+2. **Execution**: Processes stages sequentially, updates clusters within each stage (respecting maxConcurrency), enforces before-stage and after-stage tasks. Only occurs when state is `Run`
+
+When state is `Stop`, the updateRun pauses execution at the current cluster/stage and can be resumed by changing state back to `Run`
```

Comment:
I think "Stop" worth a 3rd bullet point. We can mention during stopping, we wait for the in-progress clusters to reach a deterministic state: succeeded, failed or stuck.